Message/DomainEventMessage.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace HandcraftedInTheAlps\Bundle\RedisTransportBundle\Message;
+
+class DomainEventMessage
+{
+    /**
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * @var string
+     */
+    protected $type;
+
+    /**
+     * @var string
+     */
+    protected $id;
+
+    /**
+     * @var array
+     */
+    protected $payload;
+
+    /**
+     * @var string
+     */
+    protected $created;
+
+    public function __construct(string $name, string $type, string $id, array $payload = [])
+    {
+        $this->name = $name;
+        $this->type = $type;
```

Comment:
should we add some default types `created`, `modified`, ... to constants?

Reply:
no events are implementation specific `contact.created`